tests/auth/src/index.ts

```diff
@@ -2,10 +2,17 @@ import { App } from '@microsoft/teams.apps';
 import { ConsoleLogger } from '@microsoft/teams.common/logging';
 import { DevtoolsPlugin } from '@microsoft/teams.dev';
 
-const app = new App({
-  logger: new ConsoleLogger('@tests/auth', { level: 'debug' }),
-  plugins: [new DevtoolsPlugin()],
+// :snippet-start: auth-config
+const app = new App({ 
+  oauth: { // oauth configurations
+    // the name of the auth connection to use.
+    // it should be the same as the OAuth connection name defined in the Azure Bot configuration.
+    defaultConnectionName: 'graph' 
+  },
+  logger: new ConsoleLogger('@tests/auth', { level: 'debug' }), // optional
+  plugins: [new DevtoolsPlugin()], // optional
```

Comment:
IMO let's remove it? In Oauth flows this doesn't add much value does it?

Reply:
fair, removed it